e2e_tests/src/e2e_conduit/scenarios/follower_indexer_scenario.py

```diff
@@ -0,0 +1,15 @@
+import e2e_conduit.fixtures.importers as importers
+import e2e_conduit.fixtures.processors as processors
+import e2e_conduit.fixtures.exporters as exporters
+from e2e_conduit.scenarios import Scenario
+
+
+def follower_indexer_scenario(sourcenet):
+    return Scenario(
+        "follower_indexer_scenario",
+        importer=importers.FollowerAlgodImporter(sourcenet),
+        processors=[
+            processors.BlockEvaluator(),
+        ],
```

Comment:
We shouldn't need the block evaluator with this importer

Reply:
We do for now since the importer does nothing with the retrieved delta https://github.com/algorand/indexer/blob/develop/conduit/plugins/importers/algodfollower/algodfollower_importer.go#L156